GameData/Halloween/Localization/no-no.cfg

```
@@ -0,0 +1,86 @@
+// no-no.cfg v1.1.0.0
+// Halloween (HLWN)
+// created: 05 Oct 21
+// updated: 20 Oct 2022
+
+// translated by: [@tplive](https://github.com/tplive)
+// CC BY-SA 4.0 by  zer0Kerbal
+
+Localization
+{
+	no-no
+	{
+	// ************** General
+		#HLWN-Candle-manu = Lunatic Aeronautics
+		#HLWN-kraken-manu = Dr Hoeydonck Van Kerman Kermnassus
+		#HLWN-strutCon-manu = Ungoliant Edderkopp Emporium
+
+		#HLWN-tags = hlwn (jack )lantern pumpkin halloween brown charlie eve porkjet nohark nil2work
+
+	// ************** Halloween Candle
+		#HLWN-Candle-titl-s = Halloween-stearinlys (Lite)
+		#HLWN-Candle-titl-m = Halloween-stearinlys
+		#HLWN-Candle-titl-l = Halloween-stearinlys (Stort)
+		#HLWN-Candle-desc = Stearinlys som romvesen-gresskar! Virker i verdensrommet også! Vi har gjort det elektrisk! Hold det under 1400°, for selv magiske lys smelter!
+		
+		#HLWN-Candle-on = Lys På
+		#HLWN-Candle-off = Lys Av
+		#HLWN-Candle-tog = Veksle lys på/av
+		
+		#HLWN-Candle-eon = Tenn lys
+		#HLWN-Candle-eoff = Slukk lys
+		#HLWN-Candle-etog = Veksle lys på/av
+		
+	// ************** Halloween Fuelline
+		#HLWN-Fuelline-titl = Halloween Drivstoffslange
+		#HLWN-Fuelline-desc = O Store Gresskar har en lang, kraftig og fargerik drivstoffslange for kerbalene! YEY!
+		
+	// ************** Halloween strutConnector
+		#HLWN-strutCon-titl = Ungoliants Klissete Tråd #XVII
+		#HLWN-strutCon-desc = Fordi denne slangekoblingen er laget av Kraken Edderkopp-silke, er den svært kraftig og lenger enn vanlige slanger. Den virker nesten forhekset... 
```

ONLY A SUGGESTION, both of these are valid

```suggestion
		#HLWN-strutCon-desc = Fordi denne slangekoblingen er laget av Kraken Edderkopp-silke, er den svært kraftig og lengre enn vanlige slanger. Den virker nesten forhekset... 
```